src/index.js

```diff
@@ -156,6 +156,10 @@ async function prerender (parentCompilation, request, options, inject, loader) {
       // suppress console-proxied eval() errors, but keep console proxying
       virtualConsole: new jsdom.VirtualConsole({ omitJSDOMErrors: false }).sendTo(console),
 
+      // `url` sets the value returned by `window.location`, `document.URL`...
+      // Useful for routers that depend on the current URL (such as react-router or reach-router)
+      url: options.documentUrl,
```

Comment:
Could we fall back to a default of `http://localhost` here in order to fix #14?

Reply:
I currently don't have internet, will change it once I get it fixed. Thanks for the review 😀